src/Admin/LineIns.vue

```diff
@@ -25,8 +18,8 @@ export default {
   name: 'admin-line-ins',
   data () {
     return {
-      rowsPerPage: 20,
-      searchQuery: '',
+      rowsPerPage: "0",
```

Comment:
Should be an integer.

Reply:
it was moved to custom table component only and default value is integer 0, look on "move rows and filter input to custom table component"

Reply:
So it should be either removed or given a proper default value.

Reply:
I see... i look on wrong file